Orange/base.py

```diff
@@ -114,9 +114,9 @@ def __call__(self, data):
         if len(data.domain.class_vars) > 1 and not self.supports_multiclass:
             raise TypeError("%s doesn't support multiple class variables" %
                             self.__class__.__name__)
-
         self.domain = data.domain
         model = self._fit_model(data)
+        model.data = data
```

Comment:
Can we avoid this. We already have `model.original_data` below which we would like to get rid of.
Every model should not store the complete data set.